src/main/java/bernie/exceptions/InvalidArgumentException.java

```diff
@@ -0,0 +1,11 @@
+package bernie.exceptions;
+
+/**
+ * InvalidArgumentException is thrown whenever the user enters invalid arguments into a valid command. Such examples
+ * are marking a task when it is already marked, or marking a task that doesn't exist, etc.
+ */
+public class InvalidArgumentException extends Exception {
+    public InvalidArgumentException(String errorMessage) {
+        super(errorMessage);
+    }
+}
```

Comment:
Instead of inheriting from `Exception` class, you could consider inheriting from `BernieException`. Moreover, as you add more exceptions to this package, you could also consider making all the "handmade" exception as static attribute to `BernieException` class.